src/components/EditFeatureDrawer/EditFeatureFullForm/index.tsx

```diff
@@ -0,0 +1,223 @@
+  const executeGetFeature = useExecuteGetFeature();
+
+  const [tabConfig, setTabConfig] = useState<PropertyFormTabConfig<PropertyFormItemEditConfig>[]>();
+  const [initialValues, setInitialValues] = useState<Record<string, any>>();
+  const [errorMsg, setErrorMsg] = useState<string>();
+
+  const allowedEditMode = useAppSelector(
+    state => state.editFeature.userEditMode
+  );
+
+  const [form] = useForm();
+  const map = useMap();
+  const dispatch = useAppDispatch();
+  const client = useSHOGunAPIClient();
+
+  const reloadFeature = useCallback(async (id: string) => {
+    if (!layer || !isWmsLayer(layer)) {
+      return;
+    }
+
+    const updatedFeatures = await executeGetFeature({
+      layer: layer,
+      featureId: id
+    });
+
+    if (
+      updatedFeatures?.features[0] &&
+      (allowedEditMode.includes('UPDATE') ||
+        allowedEditMode.includes('DELETE'))
+    ) {
+      dispatch(setFeature(updatedFeatures?.features[0]));
+    }
+    return;
+  }, [allowedEditMode, dispatch, executeGetFeature, layer]);
+
+  const update = useCallback(async () => {
+    if (!map || !client) {
+      return;
+    }
+
+    let editFormConfig = layer.get('editFormConfig') as PropertyFormTabConfig<PropertyFormItemEditConfig>[];
+
+    if (editFormConfig.length === 0) {
+      Logger.warn(`Layer ${layer.get('name')} has no 'editFormConfig' set`);
+      return;
+    }
+
+    const properties = cloneDeep(feature?.properties) || {};
+
+    Object.entries(properties).forEach(([key, value]) => {
+      const tabConfigs = editFormConfig?.filter(tabCfg => {
+        return tabCfg.children?.find(formCfg => formCfg.propertyName === key);
+      });
+
+      if (tabConfigs.length > 1) {
+        Logger.warn(`Property ${key} is configured in multiple tabs. Is this intended?`);
+      }
+
+      if (tabConfigs && tabConfigs[0]) {
+        const isDate = tabConfigs[0].children?.find(cfg => {
+          return cfg.propertyName === key && cfg.component === 'DATE';
+        });
+
+        if (isDate) {
+          properties[key] = moment(value);
+        }
+
+        const isUpload = tabConfigs[0].children?.find(cfg => {
+          return cfg.propertyName === key && cfg.component === 'UPLOAD';
+        });
+
+        if (isUpload) {
+          properties[key] = [{
+            name: value,
+            status: 'done'
+          }];
+        }
+      }
+    });
+
+    form.resetFields();
+    form.setFieldsValue(properties);
+
+    setTabConfig(editFormConfig);
+    setInitialValues(properties);
+  }, [map, client, layer, feature?.properties, form]);
+
+  useEffect(() => {
+    update();
+  }, [update]);
+
+  const onSaveSuccess = (responseText?: string) => {
+    if (!responseText) {
+      return;
+    }
+    setErrorMsg(undefined);
+    const parser = new DOMParser();
+    const xmlDoc = parser.parseFromString(responseText, 'text/xml');
+
+    // get feature id from response
+    const featureId = xmlDoc.getElementsByTagName('ogc:FeatureId');
```

Comment:
Is currently only WFS-T 2.0.0 supported?

Reply:
WFS Transaction is currently implemented in [1.1.0](https://github.com/terrestris/shogun-gis-client/pull/885/files/0a016436a3a23be7586a12e5589fea6f61d2dd73#diff-c1019a11bf7b99a4475f13c98e1427accd8d2f03976f4071ca5553797c7eccf5R120).